prodtests/full-system-test/dpl-workflow.sh

```diff
@@ -456,6 +456,7 @@ has_detector_matching PRIMVTX && [[ ! -z "$VERTEXING_SOURCES" ]] && add_W o2-pri
 
 if [[ $BEAMTYPE != "cosmic" ]]; then
   has_detectors_reco ITS && has_detector_matching SECVTX && [[ ! -z "$SVERTEXING_SOURCES" ]] && add_W o2-secondary-vertexing-workflow "$DISABLE_DIGIT_ROOT_INPUT $DISABLE_ROOT_OUTPUT $TPC_CORR_SCALING --vertexing-sources $SVERTEXING_SOURCES --threads $SVERTEX_THREADS --pipeline $(get_N secondary-vertexing MATCH REST $SVERTEX_THREADS SECVTX)"
+  has_detectors_reco ITS && has_detector_matching SECVTX && has_detector_matching STRK && add_W o2-strangeness-tracking-workflow "$DISABLE_MC $DISABLE_DIGIT_ROOT_INPUT $DISABLE_ROOT_OUTPUT --pipeline $(get_N strangeness-tracking MATCH REST 1 STRK)"
```

Comment:
Could you add to the https://github.com/AliceO2Group/AliceO2/blob/3715b202069ef621e60aa6bc5b5aca60d9304ce3/Detectors/GlobalTrackingWorkflow/src/strangeness-tracking-workflow.cxx#L46 `--disable-root-output` option and condition adding https://github.com/AliceO2Group/AliceO2/blob/3715b202069ef621e60aa6bc5b5aca60d9304ce3/Detectors/GlobalTrackingWorkflow/src/strangeness-tracking-workflow.cxx#L71 by this option being false.

Reply:
Thanks for spotting, done!